lib/datadog/appsec.rb

```diff
@@ -1,3 +1,4 @@
+# typed: false
```

Comment:
Should comment directives have an empty line separating them from these `require`?